### PR TITLE
Return token cost on session_end

### PR DESCRIPTION
**Pull request by sprajosh**
## 📥 Pull Request

**📘 Description**
Return token cost after session end. This can be used by the user if they want.

**🔄 Related Issue (if applicable)**
https://github.com/AgentOps-AI/agentops/issues/249

### CODE REVIEW

**Comment by areibman** [comment]
Looks great-- thanks @sprajosh!

**Review by areibman — APPROVED** [review]

File: agentops/client.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 """
-    AgentOps client module that provides a client class with public interfaces and configuration.
+AgentOps client module that provides a client class with public interfaces and configuration.
 
-    Classes:
-        Client: Provides methods to interact with the AgentOps service.
+Classes:
+    Client: Provides methods to interact with the AgentOps service.
 """
```
Comment (areibman) [inline agentops/client.py:6]: Can you please update the docstring to show that it returns the token cost as a Decimal?

Also, update the type hint

```
def end_session(...) -> Decimal:
    ...
```

**Review by areibman — CHANGES_REQUESTED** [review]

**Comment by sprajosh** [comment]
@areibman I can see some issues which have a PR linked against them but are still open. One of them is the ollama support that I added. It will be great if you can go over the issues and mark the relevant ones as resolved.

File: agentops/client.py (hunk 1 above)
Comment (sprajosh) [reply agentops/client.py:6]: Have added type hints.

Do you want me to add all methods of client to the docstring in the beginning of client.py?

File: agentops/client.py [hunk 2]
```diff
@@ -389,7 +388,7 @@ def end_session(
         end_state_reason: Optional[str] = None,
         video: Optional[str] = None,
         is_auto_end: Optional[bool] = None,
-    ):
+    ) -> Decimal:
         """
         End the current session with the AgentOps service.
```
Comment (areibman) [inline agentops/client.py:393]: Sorry-- I meant add the `returns:...` here in the signature for end_session

File: agentops/client.py (hunk 1 above)
Comment (areibman) [reply agentops/client.py:6]: No need, thanks

**Review by braelyn-ai — APPROVED** [review]
LGTM - thanks for the contribution!

**Review by areibman — APPROVED** [review]

**Comment by areibman** [comment]
@sprajosh Good to go! You're good to merge

**Pull request merged**